book/src/build_wasm.md

```diff
@@ -64,11 +66,12 @@ void main() async {
 }
 ```
 
-[^1]: When running Flutter Web, you may encounter issues with downloading Flutter
-      support scripts which have not been marked as `crossorigin="anonymous"` and
-      therefore cannot be loaded. For local testing, you can specify `credentialless`
-      instead.
+[^1]:
+    When running Flutter Web, you may encounter issues with downloading Flutter
+    support scripts which have not been marked as `crossorigin="anonymous"` and
+    therefore cannot be loaded. For local testing, you can specify `credentialless`
+    instead.
 
 [`wasm_bindgen`]: https://rustwasm.github.io/docs/wasm-bindgen
-[`Cross-Origin-Resource-Policy`]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP)
-[`Cross-Origin-Embedder-Policy`]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy
\ No newline at end of file
+[`cross-origin-resource-policy`]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP)
+[`cross-origin-embedder-policy`]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy
```

Comment:
This doesn't break the link:
https://github.com/temeddix/flutter_rust_bridge/blob/improve-formats/book/src/build_wasm.md